src/osd/modules/lib/osdlib_macosx.cpp

```diff
@@ -215,3 +219,40 @@ char *osd_get_clipboard_text(void)
 
 	return result;
 }
+
+//============================================================
+//  osd_dynamic_bind
+//============================================================
+
+osd_dynamic_bind_base::osd_dynamic_bind_base(const char *symbol, const std::vector<std::wstring> libraries)
+	: m_module(nullptr), m_function(nullptr)
+{
+	for (int i = 0; i < libraries.size(); i++)
```

Comment:
It's better to use C++11 style iteration in cases like this

    for (auto const &lib : libraries)
    ...